pkgs/development/python-modules/osmpythontools/default.nix

```diff
@@ -13,14 +13,17 @@
 
 buildPythonPackage rec {
   pname = "osmpythontools";
-  version = "0.2.8";
+  version = "0.2.9";
 
-  src = fetchPypi {
-    pname = "OSMPythonTools";
-    inherit version;
-    sha256 = "8a33adbd266127e342d12da755075fae08f398032a6f0909b5e86bef13960a85";
+  src = fetchFromGitHub {
+    owner = "mocnik-science";
+    repo = "osm-python-tools";
+    rev = "v${version}";
+    sha256 = "1qpj03fgn8rmrg9a9vk7bw32k9hdy15g5p2q3a6q52ykpb78jdz5";
   };
 
+  patches = [ ./remove-test-only-dependencies.patch ];
```

Comment:
Is there an upsteam patch we can fetch?

Reply:
Not that I know of. And I'm too unsure how test-only dependencies _shoud_ nowadays be declared in Python setuptools-based to file a PR at upstream. (`tests_require` seems to have been deprecated in [setuptools 41.5.0](https://setuptools.readthedocs.io/en/latest/history.html#v41-5-0) (pypa/setuptools#1878) and it's unclear to me what replaces it for setuptools-based projects that do not use tox.)

Reply:
Hmmm... Just keep it like this for now.